anaconda-pkg-build/linux/Dockerfile

```diff
@@ -78,6 +78,7 @@ RUN yum install -q -y deltarpm \
         util-linux \
         #wget \
         which \
+        procps \
```

Comment:
I guess the package should be named: `procps-ng` ... and the list is in alphabetic order

Reply:
done